src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -83,6 +83,12 @@ const availableActions: AvailableActionsType = {
     },
 };
 
+const generateMapAction = {
```

Comment:
> is there a reason why this couldn't be configured like the other actions above?

The other actions open a modal but I think I will try and see if I can make the a conditional statement for the MenuItem so that on click it does a different action.